src/test/java/io/stargate/sgv2/jsonapi/api/v1/VectorizeSearchIntegrationTest.java

```diff
@@ -0,0 +1,820 @@
+          .then()
+          .statusCode(200)
+          .body("data.document", is(nullValue()))
+          .body("status", is(nullValue()))
+          .body("errors", is(notNullValue()))
+          .body("errors", hasSize(1))
+          .body("errors[0].message", startsWith(ErrorCode.SHRED_BAD_VECTORIZE_VALUE.getMessage()))
+          .body("errors[0].errorCode", is("SHRED_BAD_VECTORIZE_VALUE"))
+          .body("errors[0].exceptionClass", is("JsonApiException"));
+    }
+  }
+
+  @Nested
+  @Order(3)
+  class InsertManyCollection {
+    @Test
+    public void insertVectorSearch() {
+      String json =
+          """
+        {
+           "insertMany": {
+              "documents": [
+                {
+                  "_id": "2",
+                  "name": "Logic Layers",
+                  "description": "An AI quilt to help you sleep forever",
+                  "$vectorize": "An AI quilt to help you sleep forever"
+                },
+                {
+                  "_id": "3",
+                  "name": "Vision Vector Frame",
+                  "description": "A deep learning display that controls your mood",
+                  "$vectorize": "A deep learning display that controls your mood"
+                }
+              ]
+           }
+        }
+        """;
+
+      given()
+          .header(HttpConstants.AUTHENTICATION_TOKEN_HEADER_NAME, getAuthToken())
+          .contentType(ContentType.JSON)
+          .body(json)
+          .when()
+          .post(CollectionResource.BASE_PATH, namespaceName, collectionName)
+          .then()
+          .statusCode(200)
+          .body("status.insertedIds[0]", is("2"))
+          .body("status.insertedIds[1]", is("3"))
+          .body("data", is(nullValue()))
+          .body("errors", is(nullValue()));
+
+      json =
+          """
+        {
+          "find": {
+            "filter" : {"_id" : "2"}
+          }
+        }
+        """;
+
+      given()
+          .header(HttpConstants.AUTHENTICATION_TOKEN_HEADER_NAME, getAuthToken())
+          .contentType(ContentType.JSON)
+          .body(json)
+          .when()
+          .post(CollectionResource.BASE_PATH, namespaceName, collectionName)
+          .then()
+          .statusCode(200)
+          .body("data.documents[0]._id", is("2"))
+          .body("data.documents[0].$vector", is(notNullValue()))
+          .body("errors", is(nullValue()));
+    }
+  }
+
+  public void insertVectorDocuments() {
+    String json = """
+      {
+        "deleteMany": {
+        }
+      }
+      """;
+
+    given()
+        .header(HttpConstants.AUTHENTICATION_TOKEN_HEADER_NAME, getAuthToken())
+        .contentType(ContentType.JSON)
+        .body(json)
+        .when()
+        .post(CollectionResource.BASE_PATH, namespaceName, collectionName)
+        .then()
+        .statusCode(200)
+        .body("errors", is(nullValue()))
+        .extract()
+        .path("status.moreData");
```

Comment:
What does this test? (that is, is extract + path needed here?)

Reply:
This is not test, this is for initial test data setup. It is used to delete data if there are more than paging size. Here we load only few data I have not paginated the delete.

Reply:
Ok, my question was related to extraction of response and getting `status.moreData`, which is not used. So it seems unnecessary (last 2 lines). Not a big deal, was just wondering if there was some specific purpose.